aclogview/CM_Combat.cs

```diff
@@ -161,4 +185,121 @@ public class QueryHealthResponse : Message {
             treeView.Nodes.Add(rootNode);
         }
     }
+
+    public class AttackerNotificationEvent : Message
+    {
+        public PStringChar defenders_name;
+        public uint damage_type;
+        public double severity; // 0.0 to 1.0
+        public uint damage;
+        public bool critical;
+        public ulong attack_conditions;
+
+        public static AttackerNotificationEvent read(BinaryReader binaryReader)
+        {
+            AttackerNotificationEvent newObj = new AttackerNotificationEvent();
+            newObj.defenders_name = PStringChar.read(binaryReader);
+            newObj.damage_type = binaryReader.ReadUInt32();
+            newObj.severity = binaryReader.ReadDouble();
+            newObj.damage = binaryReader.ReadUInt32();
+            newObj.critical = binaryReader.ReadBoolean();
```

Comment:
From what I can see, it looks like critical is actually a full int, not just a single byte.